Static/Row.swift

```diff
@@ -57,6 +57,8 @@ public struct Row: Hashable, Equatable {
         }
     }
 
+    public var associatedObject: Any?
```

Comment:
This can be accomplished using the `context` when creating a `Row`. Same idea, except its a dictionary so you can store multiple objects and then access them at configuration time.